Author: Samuel Mimram <smimram@debian.org>
Description: Define the package_version to avoid necessity for configure.
--- a/module/sysprof-module.c
+++ b/module/sysprof-module.c
@@ -19,6 +19,8 @@
  * Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
  */
 
+#define PACKAGE_VERSION "1.0.7-debian"
+
 #ifdef CONFIG_SMP
 # define __SMP__
 #endif
@@ -36,8 +38,6 @@
 
 #include "sysprof-module.h"
 
-#include "../config.h"
-
 #include <linux/version.h>
 
 #if LINUX_VERSION_CODE < KERNEL_VERSION(2,6,18)
